sunsynk/sunsynk.py

```diff
@@ -24,6 +25,8 @@ class Sunsynk:
     timeout: int = attrs.field(default=10)
     read_sensors_batch_size: int = attrs.field(default=60)
     timeouts: int = 0
+    dongle_serial_number: int = attrs.field(default=0)
+    allow_gap: int = 1
```

Comment:
I like the addition of allow_gap, as having this at 1 is a bit counterintuitive to what we're trying to do with grouping them (sending less data). In your case you sending less data is important

Reply:
:)